corehq/ex-submodules/couchforms/_design/filters/xforms.js

```diff
@@ -11,10 +11,12 @@ function(doc, req)
     switch (doc_type) {
         case "XFormInstance":
         case "XFormArchived":
+        case "XFormError":
         case "XFormDeprecated":
         case "XFormDuplicate":
         case "XFormInstance-Deleted":
         case "HQSubmission":
+        case "SubmissionErrorLog":
             return true;
         default:
             return false;
```

Comment:
I could be wrong, but I think that filter changes actually don't require any indexing. pretty sure @dannyroberts or @esoergel looked into that recently